python/paddle/tensor/math.py

```diff
@@ -4464,7 +4464,7 @@ def logit(x, eps=None, name=None):
             \end{array}\right.
 
     Args:
-        x (Tensor): The input Tensor with data type float32, float64.
+        x (Tensor): The input Tensor with data type uint16, float16, float32, float64.
```

Comment:
In fact, this API supports bfloat16 instead of uint16 data type. The reasons are as follows: 

https://github.com/PaddlePaddle/Paddle/blob/b4d7e1e06a2ce03ff38299bb8b8ebf959ee4bc3d/python/paddle/fluid/data_feeder.py#L115-L119

Reply:
should I change it into bfloat16?